packages/core-modules/test/contracts/modules/CoreElectionModule.test.js

```diff
@@ -347,13 +347,22 @@ describe('CoreElectionModule Setup, Getters, Setters and Voting', () => {
       assertBn.eq(await ElectionStorageMock.getVoterVoteVotePowerMock(voters[4].address), 100);
     });
 
-    describe('when attempting to vote again', () => {
-      it('reverts', async () => {
-        await assertRevert(
-          CoreElectionModule.connect(voters[1]).elect([candidates[0].address], [0]),
-          'AlreadyVoted'
+    describe('when casting a vote again', () => {
```

Comment:
Inside this `describe`, I would cast 2 votes, so the test contains itself and is not dependant on running with after the previous one. E.g.: if you run this with `describe.only` it would fail or the test wouldn't be testing the double vote.

Reply:
FYI, this is just an opinion I think we should start to think about, we are doing this same way of testing on other places, this PR is not the place to implement the fix.

Reply:
Like it. Added a ticket for that #636